### PR TITLE
FOLLOW-1337: Disable FORCE_CALL_STRATEGY for 10% of sessions

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

We want to re-enable the update calls that are disabled via FORCE_CALL_STRATEGY.
Because this may result in higher subnet load, we want to first roll this out for 10% of sessions.

# Changes

1. Initialize `FORCE_CALL_STRATEGY` on app load, to `"query"` 90% of the time and to `undefined` 10% of the time.
2. Output what it was initialized to, in case we need to know this later for trouble shooting.

# Tests

1. Manually tested by checking the logged value and by adding additional logging in `queryAndUpdate`.
2. `FORCE_CALL_STRATEGY` itself has existing test coverage so it should work either way.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/constants/mockable.constants.ts [hunk 1]
```diff
@@ -5,7 +5,33 @@
...
+//   0: means we still do only query calls in these places.
+// 100: means we do query+update calls in all these places.
+//  10: means we do query+update calls in these places for 10% of sessions.
+//
+// Whether we do query+update calls remains fixed for the duration of the
+// sessions, by which we mean until the next time the app/page is reloaded.
+const RESTORE_QUERY_AND_UPDATE_ROLLOUT_PERCENTAGE: number = 10;
+
+const getForceCallStrategyForPercentage = (
+  percentage: number
```
Comment (mstrasinskis) [inline frontend/src/lib/constants/mockable.constants.ts:21]: What do you think about renaming `percentage` to `queryAndUpdatePercentage`?

File: frontend/src/lib/constants/mockable.constants.ts [hunk 2]
```diff
@@ -5,7 +5,33 @@
...
+
+const initForceCallStrategy = (): undefined | "query" => {
+  const strategy = getForceCallStrategyForPercentage(
+    RESTORE_QUERY_AND_UPDATE_ROLLOUT_PERCENTAGE
+  );
+  console.info("FORCE_CALL_STRATEGY is initialized to:", strategy);
+  return strategy;
+};
+
+export const FORCE_CALL_STRATEGY: "query" | undefined = initForceCallStrategy();
```
Comment (mstrasinskis) [inline frontend/src/lib/constants/mockable.constants.ts:34]: Would it make sense to stick to a single call strategy for e2e tests?

**Review by mstrasinskis — APPROVED** [review]

File: frontend/src/lib/constants/mockable.constants.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/constants/mockable.constants.ts:34]: Or make it 50%.

Comment (dskloetd) [reply frontend/src/lib/constants/mockable.constants.ts:34]: Actually, that's not a bad idea. Unfortunately there is a bit of flakiness in the e2e tests as a result (which I'm trying to fix at the same time) and I'm not sure what would be the best way to accomplish this. Maybe with a feature flag? I'm going to think about it!

File: frontend/src/lib/constants/mockable.constants.ts (hunk 1 above)
Comment (dskloetd) [reply frontend/src/lib/constants/mockable.constants.ts:21]: I think for a 1 line function with only a single percentage, the shorter name is more readable. The constant that's passed in already has the super long name for maximum precision.
`queryAndUpdatePercentage` still wouldn't tell you what exactly the percentage means.

Comment (mstrasinskis) [reply frontend/src/lib/constants/mockable.constants.ts:21]: You’ve convinced me, thanks.

File: frontend/src/lib/constants/mockable.constants.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/constants/mockable.constants.ts:34]: I like the feature flag idea.

**Pull request merged**